### PR TITLE
Popup embellished !

### PR DESCRIPTION
**Pull request by AlexisCatrix**

### CODE REVIEW

**Review by gaetan-cordonnier — DISMISSED** [review]
Ok for me !

**Review by ThomasDruart — DISMISSED** [review]
Ok for me

**Comment by LoicBrassart** [comment]
Everytime you commit, you snap existing reviews
-> I shouldn't be assigned here, there isn't the mandatory validation yet ;-)

**Comment by LoicBrassart** [comment]
:arrow_up: Misclick, sorry about that !

**Review by ThomasDruart — DISMISSED** [review]
Good

File: src/components/MapLille.jsx [hunk 1]
```diff
@@ -53,11 +126,49 @@ class MapLille extends React.Component {
...
               ]}
             >
-              <Popup>
-                {station.fields.nom}
-                <br />
-                {station.fields.adresse}
-              </Popup>
+              {station.fields.etat.includes("HORS SERVICE") ? (
+                ""
+              ) : (
```
Comment (LoicBrassart) [inline src/components/MapLille.jsx]: You could have used `&&` instead of `?:`

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by gaetan-cordonnier — APPROVED** [review]
Good

**Review by ThomasDruart — APPROVED** [review]
ok

**Pull request merged**